resources/sass/pages/_committee.scss

```diff
@@ -0,0 +1,127 @@
+.ris-committee {
+    .ris-section-wrapper {
+        border: 0;
+    }
+    &__wrapper {
+        display: flex;
+        margin-bottom: 36px;
+        @media only screen and (max-width : 767px) {
```

Comment:
in other places i've seen that `breakpoint` mixin/function was used, is there a reason why not use it here?